### PR TITLE
[GUI] Sexier log in screen

### LINKED ISSUE #1130: Update login screen (alternative to combo box)
Currently, the user chooses the device he wants to log into using a combo box.

While it's ugly, it's also useless most of the time since users will rarely have more than one device per machine. In addition, the combo box does not let you see precisely which organization/device you are login into.

To replace it, devices should be displayed as a list (in the same way users are listed). When a device is selected, the prompt for the password appears. This allows us to directly prompt for the password if only one device is found on the machine.

### PR DESCRIPTION
**Pull request by Max-7**
Closes #1130

### CODE REVIEW

**Comment by bitlogik** [comment]
Does it fix https://github.com/Scille/parsec-cloud/issues/1169 ?

File: tests/core/gui/test_workspaces.py [hunk 1]
```diff
@@ -30,7 +29,8 @@ async def test_add_workspace(
     monkeypatch.setattr(
         "parsec.core.gui.workspaces_widget.get_text_input", lambda *args, **kwargs: (workspace_name)
     )
-    await aqtbot.mouse_click(w_w.button_add_workspace, QtCore.Qt.LeftButton)
+    async with aqtbot.wait_signal(w_w.list_success, timeout=2000):
+        await aqtbot.mouse_click(w_w.button_add_workspace, QtCore.Qt.LeftButton)
```
Comment (vxgmichel) [inline tests/core/gui/test_workspaces.py:33]: Why not use `aqtbot.wait_until`? @touilleMan and I noticed it to be more robust than `wait_signal`?

File: tests/core/gui/test_workspaces.py [hunk 2]
```diff
@@ -65,7 +64,8 @@ async def test_rename_workspace(
 
     # Create a workspace and make sure the workspace is displayed
     core = logged_gui.test_get_core()
-    await core.user_fs.workspace_create("Workspace1")
+    async with aqtbot.wait_signal(w_w.list_success, timeout=2000):
+        await core.user_fs.workspace_create("Workspace1")
```
Comment (vxgmichel) [inline tests/core/gui/test_workspaces.py:68]: Same as https://github.com/Scille/parsec-cloud/pull/1193/files#r468053106

File: tests/core/gui/test_workspaces.py (hunk 1 above)
Comment (Max-7) [reply tests/core/gui/test_workspaces.py]: I forgot that wait_until also as a timeout parameter. It's been fixed.

File: tests/core/gui/test_workspaces.py (hunk 2 above)
Comment (Max-7) [reply tests/core/gui/test_workspaces.py]: Fixed.

**Pull request merged**